src/main/java/org/opentestsystem/delivery/logging/EventInfo.java

```diff
@@ -82,20 +96,52 @@ public EventInfo withCheckpoint(String checkpoint) {
    * @param message new value of message
    * @return Copy of this EventInfo with a new message
    */
-  public EventInfo withMessage(String message) {
-    return toBuilder().message(Optional.of(message)).build();
+  public EventInfo withMessage(final String message) {
+    return toBuilder().message(message).build();
   }
 
-  @AutoValue.Builder
-  public abstract static class Builder {
-    public abstract Builder event(String event);
-
-    public abstract Builder checkpoint(Optional<String> checkpoint);
-
-    public abstract Builder message(Optional<String> message);
-
-    public abstract Builder data(Map<EventData, Object> data);
-
-    public abstract EventInfo build();
+  public static class EventInfoBuilder {
```

Comment:
After fighting with the AutoValue builder for a while, trying to get it to accept a null event and convert it to an empty string I realized I'd spent more time fighting with a convenience tool than using it.  ...so, I replaced the AutoValue with a standard Builder implementation.